modules/host/network_test.go

```diff
@@ -73,9 +73,12 @@ func TestHostWorkingStatus(t *testing.T) {
 	}
 	defer ht.Close()
 
-	if ht.host.WorkingStatus() != modules.HostWorkingStatusChecking {
-		t.Fatal("expected working state to initially be modules.HostWorkingStatusChecking")
-	}
+	// this causes an ndf, because it relies on the host tester starting up and
+	// fully returning faster than the first check, which isnt always the case.
+	// Disabled for now.
+	// if ht.host.WorkingStatus() != modules.HostWorkingStatusChecking {
+	// 	t.Fatal("expected working state to initially be modules.HostWorkingStatusChecking")
+	// }
```

Comment:
This is where we could insert a disrupt dependency to make sure that the check doesn't run until the host is full started up, and to also make sure that this check doesn't happen until the first check has run. Since this is all inside of the host package, we can do this pretty easily.